.pre-commit-config.yaml

```
@@ -0,0 +1,178 @@
+        files: \.env$
+      - id: create-session-commit
+        name: "commit-msg: Create session commit message"
+        entry: scripts/prepare-commit-msg/create_session_commit.py
+        language: python
+        stages: [commit-msg]
+      - id: pytest-fast
+        name: "python: Run fast tests"
+        entry: pytest
+        language: python
+        types: [python]
+        args: ["-m", "not slow"] # Assumes you mark slow tests
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)